cf-monitord/env_monitor.c

```diff
@@ -640,15 +640,7 @@ static void ArmClasses(EvalContext *ctx, const Averages *const av)
                 strcat(ldt_buff, buff);
             }
 
-            if (CF_THIS[i] > av->Q[i].expect)
-            {
-                snprintf(buff, CF_BUFSIZE, "%s_high_ldt", name);
-            }
-            else
-            {
-                snprintf(buff, CF_BUFSIZE, "%s_high_ldt", name);
```

Comment:
@nickanderson AFAICT this was intended to be setting the `_low_ldt` class instead. If greater -> high, if lower -> low.

Reply:
Ah, okay. Here is the better fix then.

https://github.com/cfengine/core/pull/5033